osgp/protocol-adapter-dlms/osgp-dlms/src/main/resources/dlmsprofiles/dlmsprofile-dsmr22.json

```diff
@@ -98,6 +98,43 @@
           "valuetype": "DYNAMIC",
           "value": "OK",
           "access": "R"
+        },
+        {
+          "id": 5,
+          "description": "daylights_savings_begin",
+          "datatype": "octet-string",
+          "valuetype": "SET_BY_CLIENT",
+          "access": "RW"
+        },
+        {
+          "id": 6,
+          "description": "daylights_savings_end",
+          "datatype": "octet-string",
+          "valuetype": "SET_BY_CLIENT",
+          "access": "RW"
+        },
+        {
+          "id": 7,
+          "description": "daylights_savings_deviation",
+          "datatype": "integer",
+          "valuetype": "SET_BY_CLIENT",
```

Comment:
This attribute can't be written by the client. It's fixed in the profile. Same applies for other protocol versions.

Reply:
valuetype changed to FIXED_IN_PROFILE